bbb-common-web/src/main/java/org/bigbluebutton/presentation/handlers/AbstractCommandHandler.java

```diff
@@ -51,23 +51,28 @@ public void onStart(NuProcess nuProcess) {
 
   @Override
   public void onStdout(ByteBuffer buffer, boolean closed) {
-    if (buffer != null) {
+    if (buffer != null && buffer.remaining() > 0) {
       CharBuffer charBuffer = StandardCharsets.UTF_8.decode(buffer);
-      stdoutBuilder.append(charBuffer);
+      String chunk = charBuffer.toString();
+      stdoutBuilder.append(chunk);
+      log.debug("[{}] {}", getIdTag(), chunk.trim());
     }
   }
 
   @Override
   public void onStderr(ByteBuffer buffer, boolean closed) {
     if (buffer != null) {
       CharBuffer charBuffer = StandardCharsets.UTF_8.decode(buffer);
-      stderrBuilder.append(charBuffer);
+      String chunk = charBuffer.toString();
+      stderrBuilder.append(chunk);
+      log.error("[{}] {}", getIdTag(), chunk.trim());
```

Comment:
```suggestion
      String trimmed = chunk.trim();
      if (!trimmed.isEmpty()) {
        log.error("[{}] {}", getIdTag(), trimmed);
      }
```

It is always logging error for me, even when it's blank:

![image](https://github.com/user-attachments/assets/accee3d0-9435-495d-a55f-70daac8c3af1)